app/power.c

```diff
@@ -81,9 +81,13 @@ static void power_task(void *p_arg)
 	uint16_t adc_values[NUM_OF_ADC_CHANNELS];
 
 	for (;;) {
-		xQueueReceive(SELF.queue_handle, adc_values, portMAX_DELAY);
-
-		/* TODO: Do stuff with the values */
+		uint32_t vref;
+		xQueueReceive(SELF.queue_handle, &item, portMAX_DELAY);
```

Comment:
`item` struct is missing.

Also it would be nice if you put this in a `#if 0` block, otherwise it will spam for everyone :).

Reply:
Yeah, it should just be reverted back to adc_values.
I was testing out some stuff to verify it wasn't behaving unexpected.
OK, I can add #if 0.